src/System.Private.Uri/tests/FunctionalTests/UriIpHostTest.cs

```diff
@@ -234,6 +234,34 @@ private void ParseBadIPv4Address(string badIpv4String)
 
         #region IPv6
 
+        [Theory]
+        [InlineData("fe80::e077:c9a3:eeba:b8e9", "%18")]
+        [InlineData("Fe80::e077:c9a3:eeba:b8e9", "%18")]
+        [InlineData("fE80::e077:c9a3:eeba:b8e9", "%18")]
+        [InlineData("FE80::e077:c9a3:eeba:b8e9", "%18")]
+        [InlineData("FE80::e077:c9a3:eeba:b8e9", "%eth10")]
+        [InlineData("FE80::e077:c9a3:eeba:b8e9", "")]
+        [InlineData("FE80::e077:c9a3:eeba:b8e9", "%")]
+        [InlineData("fe80::e077:c9a3:eeba:b8e9", "%\u30AF\u20E7")]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework)]
+        public void Host_IPv6LinkLocalAddress_HasScopeId(string address, string zoneIndex)
+        {
+            string scopedLiteralIpv6 = "[" + address + zoneIndex + "]";
```

Comment:
"scopedLiteralIpv6" -> "scopedLiteralIpv6Brackets" to be more consistent with variable naming in the tests above.